client/apiversion.go

```diff
@@ -196,18 +200,29 @@ func (c *authenticatingClient) getAPIVersions(serviceCatalogURL string) (*apiURL
 			http.StatusMultipleChoices,
 		},
 	}
+	apiURLVersionInfo := &apiURLVersion{
+		rootURL:          *url,
+		serviceURLSuffix: strings.Join(pathParts, "/"),
+	}
 	if err := c.sendRequest("GET", baseURL, c.Token(), requestData); err != nil {
+		if gooseerrors.IsNotFound(err) {
+			// Some OpenStack clouds do not support the
```

Comment:
Rather than comment, can we trace log here instead? Urk, actually we're still on the stdlib logging package so there's no default suppressed log method.